zappa/wsgi.py

```diff
@@ -19,31 +19,70 @@ def create_wsgi_request(
     binary_support=False,
     base_path=None,
     context_header_mappings={},
+    event_version="1.0",
```

Comment:
from the caller it appears that "version" is contained in `event_info` and is not necessary here.

```event["version"]```